scripts/release-binary.sh

```diff
@@ -117,6 +117,9 @@ if [ -n "${DST}" ]; then
   gsutil cp "${BINARY_NAME}" "${SHA256_NAME}" "${DST}/"
 fi
 
+# Build the docker image with the release binary
+bazel run ${BAZEL_BUILD_ARGS} --config=release //tools/docker:envoy
```

Comment:
If you don't want to publish it in this PR, then maybe comment it out for the time being, since it's going to be built for each merged change, and it seems wasteful if we're not going to publish the resulting artifact.

Reply:
Ok, added a TODO.

Reply:
Erm, I meant that you should comment out the `bazel run` step, until those images are published.